packages/core-app-api/api-report.md

```diff
@@ -379,25 +380,11 @@ export type FlatRoutesProps = {
 };
 
 // @public
-export class GithubAuth implements OAuthApi, SessionApi {
-  // (undocumented)
-  static create(options: OAuthApiCreateOptions): GithubAuth;
-  // (undocumented)
-  getAccessToken(scope?: string, options?: AuthRequestOptions): Promise<string>;
-  // (undocumented)
-  getBackstageIdentity(
-    options?: AuthRequestOptions,
-  ): Promise<BackstageIdentityResponse | undefined>;
-  // (undocumented)
-  getProfile(options?: AuthRequestOptions): Promise<ProfileInfo | undefined>;
+export class GithubAuth {
   // (undocumented)
+  static create(options: OAuthApiCreateOptions): typeof githubAuthApiRef.T;
```

Comment:
Not a breaking change here because `typeof githubAuthApiRef.T` is the same as the previous instance type.